.github/scripts/notifyAboutIconRequest.js

```diff
@@ -0,0 +1,31 @@
+// If the "calcite-ui-icons" label is added to an issue, generates a notification to the Icons team leads to review
+// The secret is formatted like so: icon-team-member-1, icon-team-member-2, icon-team-member-3
+// Note the script automatically adds the "@" character in to notify the icon team lead(s)
+
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+module.exports = async ({ github, context }) => {
+  const { repo, owner } = context.repo;
+
+  const payload = /** @type {import('@octokit/webhooks-types').IssuesLabeledEvent} */ (context.payload);
+  const {
+    issue: { number },
+  } = payload;
+
+  const { ICON_LEADS } = process.env;
```

Comment:
this environment variable needs to be the same thing you called it in the workflow

```suggestion
  const { ICONS_TEAM } = process.env;
```